fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java

```diff
@@ -176,6 +177,11 @@ public void run() throws JobException {
             this.relation = MTMVPlanUtil.generateMTMVRelation(mtmv, ctx);
             beforeMTMVRefresh();
             if (mtmv.getMvPartitionInfo().getPartitionType() != MTMVPartitionType.SELF_MANAGE) {
+                MTMVRelatedTableIf relatedTable = mtmv.getMvPartitionInfo().getRelatedTable();
```

Comment:
The verification here should be the case of partition column or partition type transformation